tools/tokenserver/database.py

```diff
@@ -655,13 +659,13 @@ def get_best_node(self):
         """Returns the 'least loaded' node currently available, increments the
         active count on that node, and decrements the slots currently available
         """
```

Comment:
```suggestion
        """Simply returns the Spanner node (if set), otherwise returns the 'least
        loaded' node currently available, increments the active count on that node,
        and decrements the slots currently available
        """
```